ChecksFile.wl

```diff
@@ -100,7 +103,7 @@ cma[k_,\[Mu]ch_,a_,w_]:=cm[k,\[Mu]ch,w]/.{rb-> 0.,l-> 0.,r-> (1+a),lb-> -(1-a)};
 
 
 
-(* ::Subsection::Closed:: *)
+(* ::Subsection:: *)

```

Comment:
In all commits, it would be very helpful to remove non-significant changes before committing. Don't go back and fix it, but try to do so in future commits.

Reply:
I'm not sure I understand what you mean here. Could you elaborate?

Reply:
Whitespaces and changes from "Subsection::Closed" to "Subsection" (i.e. just reflecting whether the little drop down arrow has been clicked) don't amount to meaningful changes. It is totally fine to commit changes like this, but in a separate "whitespace/formatting only" commit. This just helps tracking down changes much easier.